lib/beamchmark/formatters/html/templates.ex

```diff
@@ -65,7 +65,7 @@ defmodule Beamchmark.Formatters.HTML.Templates do
         Enum.map_join(cpu_snapshots, ", ", fn %{cpu_usage: _cpu_usage, average_all_cores: avg} ->
           format_float(avg)
         end),
-      time: Enum.map_join(1..length(cpu_snapshots), ", ", fn el -> el end)
+      time: Enum.map_join(cpu_snapshots, ", ", fn snapshot -> snapshot.timestamp end)
```

Comment:
NIT: ` Enum.map_join(cpu_snapshots, ", ", & &1.timestamp)` is much shorter (same for lines 96 and 110), but I leave the choice to you.